sergey.duev/task_4/src/main/java/com/hillel/BankConfiguration.java

```diff
@@ -0,0 +1,57 @@
+package com.hillel;
+
+import com.hillel.models.Bank;
+import com.hillel.models.BankAccount;
+import com.hillel.models.Statement;
+import com.hillel.services.ProdBankAccountService;
+import com.hillel.services.TestBankAccountService;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.EnableMBeanExport;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+@Configuration
+@ComponentScan("com.hillel.models")
+public class BankConfiguration {
+
+    @Bean
+    public ProdBankAccountService getProdBankAccountService(Bank bank) {
+        return new ProdBankAccountService(bank.getStatements());
+    }
+
+    @Bean
+    public TestBankAccountService getTestBankAccountService(Bank bank) {
+        return new TestBankAccountService(bank.getStatements());
+    }
+
+    @Bean
+    public List<BankAccount> initBankAccounts() {
```

Comment:
It's not good idea to create bean for List<BankAccount>, Better to create separate class like bankDataInitializer

And then it can be used as bean